pytest.ini

```diff
@@ -0,0 +1,20 @@
+[pytest]
+addopts =
+    # show 10 slowest invocations:
+    --durations=10
+
+    # a bit of verbosity doesn't hurt:
+    -v
+
+    # report all the things == -rxXs:
+    -ra
+
+    # show values of the local vars in errors:
+    --showlocals
+asyncio_mode = auto
+filterwarnings =
+    error
+    ignore:unclosed transport <_SelectorSocketTransport:ResourceWarning
+    ignore:unclosed <socket.socket:ResourceWarning
```

Comment:
@thehesiod If there's any chance you can figure out these warnings, that would be great. They only occur in 1 test, and for some reason don't occur on Python 3.7. But, I can't seem to find any reason there would be unclosed sockets/transports...

Reply:
we now have a helper: @jayzfbn any luck on these?  if not I can work on it

Reply:
Thanks, you can just remove those 2 ignore lines and run the tests to reproduce. I'm suspicious that the cancelling behaviour that is being tested in that test is not quite working correctly, and that is resulting in those warnings.

Reply:
I haven't been able to reproduce this.  The latest master branch gives me a clean 52 tests pass report even with no filterwarnings.  What test is failing?  Is it still reproducible?

Reply:
Just run it in CI, fails on Python 3.8, 3.9 and 3.10.
https://github.com/aio-libs/aiomcache/pull/302